### DISCUSSION TITLE
Prompt Changelog Management

### DISCUSSION THREAD

**Discussion by alexander-abramovich** [langfuse/langfuse#4316] [Ideas]
### Describe the feature or potential improvement

We save one version after another, but we’re struggling with tracking changes effectively. To maintain a clear changelog for prompt updates it would be great to have an option to submit what changed and to view differences in a Git-style comparison. How do you

**Comment by marcklingen** [comment]
Both completed
- https://langfuse.com/changelog/2025-01-28-prompt-commit-messages
- https://langfuse.com/changelog/2025-01-22-track-changes-between-prompt-versions
